packages/blockchain-link/src/index.ts

```diff
@@ -8,17 +8,17 @@ import type * as ResponseTypes from '@trezor/blockchain-link-types/lib/responses
 import type * as MessageTypes from '@trezor/blockchain-link-types/lib/messages';
 import type { Events } from '@trezor/blockchain-link-types/lib/events';
 
-const workerWrapper = (factory: BlockchainSettings['worker']): Worker => {
+const workerWrapper = (factory: BlockchainSettings['worker']): Worker | Promise<Worker> => {
```

Comment:
What's the reason for these changes?

Reply:
That worker code is now chunked and loaded dynamically in desktop app: https://github.com/trezor/trezor-suite/pull/9552/files#diff-7716251d3a5873ffcd11f408f34b57a364fb2cdabaa1c4fb433c1ce52a5284cd